src/components/WheelPicker/WheelPicker.driver.tsx

```diff
@@ -11,10 +11,12 @@ export const WheelPickerDriver = (props: ComponentProps) => {
     renderTree: props.renderTree,
     testID: `${props.testID}.list`
   }));
-  
-  const moveToItem = (index: number, numberOfRows: number, itemHeight: number) => {
+
+  const moveToItem = (index: number,
+    numberOfRows: number = NUMBER_OF_VISIBLE_ROWS,
```

Comment:
This is not the number of visible rows, this is the number of rows i.e. props.items.length

Reply:
fixed it, WDYT about exporting the `ITEMS_HEIGHT` ?
do we need this small export ?
in the WheelPicker testing you have custom values